phpBB/assets/javascript/editor.js

```diff
@@ -389,11 +389,32 @@ function getCaretPosition(txtarea) {
 	return caretPos;
 }
 
-/**
-* Allow to use tab character when typing code
-* Keep indentation of last line of code when typing code
-*/
 (function($) {
+	function handle_mentions(txtarea) {
+		var $mentionParams = $('#mention_params'),
+			mentionURL = $mentionParams.data('mentionUrl'),
+			mentionNamesLimit = $mentionParams.data('mentionNamesLimit'),
+			mentionTopicId = $mentionParams.data('topicId');
+		$(txtarea).atwho({
+			at: "@",
+			displayTpl: function(data) {
+				var avatar = (data.avatar.src) ? "<img src='" + data.avatar.src + "'>" :
```

Comment:
Why not use es6 template structure here https://caniuse.com/#search=Template%20literal